README.rst

```diff
@@ -132,3 +140,61 @@ Deployment with Dokku
 
 Alternatively, you can deploy the project using Dokku. See the
 `Caktus developer docs <http://caktus.github.io/developer-documentation/dokku.html>`_.
+
+How the license translation is implemented
+------------------------------------------
+
+First, note that translation is done in two different ways. Most things use the built-in
+Django translation support. But the translation of the actual legal text of the licenses
+is handled separately, and that's what this section is about.
+
+Second note: the initial implementation focuses on the 4.0 by-*
+licenses. Others will be added as time allows.
+
+The translation data consists of ``.po`` files, and they are managed in a separate
+repository from this code, ``https://github.com/creativecommons/cc-licenses-data``.
+This is typically checked out beside the ``cc-licenses`` repo, but can be put anywhere by changing the Django ``TRANSLATION_REPOSITORY_DIRECTORY`` setting,
+or setting the ``TRANSLATION_REPOSITORY_DIRECTORY`` environment variable.
+
+The .po files do *not* use the English messages as the message IDs, which would be
+the usual practice. FIXME: FILL IN THE REASON HERE. OR USE THE ENGLISH MESSAGES
```

Comment:
I think we got an answer from TR about this paragraph. English is the default internally; but anything user facing should be language agnostic as to the default. Can we re-word this paragraph and remove the `FIXME`